### DISCUSSION TITLE
Google Task sync with Thunderbird

### DISCUSSION THREAD

**Discussion by carterbrandon** [kewisch/gdata-provider#610] [Support] [answered]
When I flag an item on a Google Task list as 'Completed' (in a Google interface) then synchronise with Thunderbird, the completion flag stays unchanged in Thunderbird unless I've also done an edit to the task text, or unsubscribed/resubscribed the whole calendar, or if I Complete it on Thunderbird too, then sync and either Submit or Discard the change as prompted.

I was hoping that when Google Tasks got combined/integrated with calendar Reminders a few weeks ago, it might have made a difference to this, but that doesn't seem to be the case. Is this a long-term known issue?

I'm using Provider 102.0.3 and Thunderbird 102.12.0 on Windows 10.

**Comment by kewisch** [comment]
Can you provide any errors or relevant messages from the error console (Tools Menu → Error Console), with debugging enabled:
    * Open your application preferences (Tools or Edit Menu)
    * Select the `Advanced` icon at the top.
    * Click on the `General` tab, below the icons.
    * Select `Advanced Configuration` or `Config Editor`, whichever is available
    * Right Click on an empty space in the list of preferences
    * Select `New` → `Boolean`
    * Enter `calendar.debug.log` as the preference name
    * Pick `true` as the Boolean value.
    * Repeat the same for `calendar.debug.log.verbose`

  To be sure you are getting all messages it is advisable to restart the application after setting the preference.

**Reply by carterbrandon** [reply, marked as answer]
This was such an easy win! Simply turning the debug logging on in the config editor as you suggested, made it work!

**Reply by kewisch** [reply]
That is...strange, but I'm glad it works? ;-)
